### DISCUSSION TITLE
Is there a way to mirror the screen on a new monitor attached instead of display extension?

### DISCUSSION THREAD

**Discussion by Animeshz** [herbstluftwm/herbstluftwm#1597] [General]
As is :)

**Comment by t-wissmann** [comment]
Yes, but that's simply up to `xrandr`. (Or maybe more intuitively using tools like `arandr`). But mirroring the screen is entirely independent of the X window manager.

**Reply by Animeshz** [reply]
Thanks I'll try that... I knew something like that exists earlier but completely slipped out of my mind since I'm little far from monitors for a long time.
